Exoft.Gamification.Api.Common/Helpers/AutoMapperProfile.cs

```diff
@@ -1,19 +1,33 @@
 ﻿using AutoMapper;
+using Exoft.Gamification.Api.Common.Models;
 using Exoft.Gamification.Api.Common.Models.Achievement;
 using Exoft.Gamification.Api.Data.Core.Entities;
+using Exoft.Gamification.Api.Data.Core.Helpers;
 
 namespace Exoft.Gamification.Api.Common.Helpers
 {
     public class AutoMapperProfile : Profile
     {
         public AutoMapperProfile()
         {
+            // map to model
             CreateMap<Achievement, ReadAchievementModel>()
                 .ForMember(d => d.IconId, o => o.MapFrom(s => s.Icon.Id));
 
-            CreateMap<CreateAchievementModel, Achievement>();
+            CreateMap<ReturnPagingInfo<Achievement>, ReturnPagingModel<ReadAchievementModel>>()
```

Comment:
no need in both ReturnPagingModel<> and ReturnPagingInfo<>... one would be enough